src/main/java/org/mitre/synthea/engine/Module.java

```diff
@@ -66,7 +67,7 @@ public class Module implements Cloneable, Serializable {
   private static final Map<String, ModuleSupplier> modules = loadModules();
 
   private static Map<String, ModuleSupplier> loadModules() {
-    Map<String, ModuleSupplier> retVal = new ConcurrentHashMap<>();
+    Map<String, ModuleSupplier> retVal = new TreeMap<>();
```

Comment:
Note to reviewers: this change looks scary because it's no longer "concurrent", but we were never actually writing anything to the map outside of this function in regular usage.  I wanted to replace the `return retVal;` at the end with `return Collections.unmodifiableMap(retVal);` to make that obvious, but it did break unit tests that modify the map for a specific test reason.